record-and-playback/notes/scripts/notes.nginx

```diff
@@ -0,0 +1,22 @@
+#
+# BigBlueButton open source conferencing system - http://www.bigbluebutton.org/
+#
+# Copyright (c) 2019 BigBlueButton Inc. and by respective authors (see below).
+#
+# This program is free software; you can redistribute it and/or modify it under the
+# terms of the GNU Lesser General Public License as published by the Free Software
+# Foundation; either version 3.0 of the License, or (at your option) any later
+# version.
+#
+# BigBlueButton is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
+#
+
```

Comment:
Since `bbb-etherpad` package already uses a file named `notes.nginx` something has to change. The one that makes more sense to me is keeping this as `notes` and renaming the other to `pad` or `etherpad`. What do you think, @ffdixon ?

Reply:
Done!  The notes.nginx file has been renamed to etherpad.nginx in the latest build of 2.2-beta.